etc/linux-systemd/system/syncthing-relay.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Syncthing - Relay
+After=network.target
+ 
+[Service]
+User=relaysrv
```

Comment:
It's the user when we follow the documentation (https://docs.syncthing.net/users/strelaysrv.html#setting-up)
(I use /usr/local/etc in place of /etc because it's not install with a package, but we can replace it with /etc/ if you prefer).